chihacknight/decarbonize-my-state
# remove package-lock.json

## Overview

`package-lock.json` is unnecessary as we have a `yarn.lock` file. Having the file present has caused installation issues for us.

According to the [yarn documentatio](https://yarnpkg.com/features/protocols#git)n, it will use a `package-lock.json` file if present. Ours was out of sync with our `yarn.lock` and `package-lock.json` files and was missing the `scrollspy` dependency.

Closes #70 

## Testing Instructions

* run `docker-compose down --volumes` to delete your existing docker containers
* then run `docker-compose up --build` to rebuild everything. this command should run without errors and you should be able to view the site locally
